pubsublite/admin.go

```diff
@@ -29,6 +29,18 @@ var (
 	errNoSubscriptionFieldsUpdated = errors.New("pubsublite: no fields updated for subscription")
 )
 
+// BacklogLocation refers to a location with respect to the message backlog.
```

Comment:
This is used as a bool (`!= Beginning`). Should we expose it as a bool? Or will there possibly be other locations in the future?

Reply:
We want to expose this feature to users as an option between Beginning/End for consistency with the admin 'seek' operation that will soon be available.  For the seek operation, users will also have an option between BacklogLocation beginning/end.